packages/hardhat-create2-proxy/docs/decisions/003-create2proxy-salt-strategy.md

```diff
@@ -0,0 +1,76 @@
+# ADR 0003: CREATE2 Proxy Salting Strategy
+
+## Status
+
+Accepted
+
+## Date: 19-05-2025
+
+## Context
+
+`CREATE2` allows contracts to be deployed at deterministic addresses based on four inputs: the proxy address, the salt, the bytecode hash, and the initialization code. This enables predictable contract addresses, which is useful for many deployment patterns.
```

Comment:
```suggestion
`CREATE2` allows contracts to be deployed at deterministic addresses based on three inputs: the deployer (factory) address, the salt, and the initialization code hash. This enables predictable contract addresses, which is useful for many deployment patterns.
```

Reply:
the bytecode hash is the initialization code hash. They are one and the same.

Reply:
I for got about the 0xFF constant, which is the first param. I modified my suggestion